docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,296 @@
+
+```shell script
+git push
+```
+which pushes our code to the master branch.
+
+
+We should already be logged into Heroku because we deployed our frontend code but if we're not, then we have to login again using:
+```shell
+heroku login
+```
+
+After logging in we run:
+
+```shell
+heroku create
+```
+which creates a new project on Heroku that represents our backend
+
+```shell
+git push heroku master
+```
+This will push our code to the Heroku server. Again, if we want to deploy a version that is not in our master branch we run the command:
+```shell
+git push heroku <branch_name>:master
+```
+This will push our designated git branch but Heroku pretends that this branch is the master branch.
+
+After pushing our code to the Heroku server we should be given a URL. This URL is now the URL that our frontend needs to make request to.
+To verify that our backend deployed successfully, we again run the command:
+```shell
+heroku open
+```
+which should open a new browser window with a message at the top saying <strong> Hello Jumpstarters! </strong>
+
+Now that our backend and frontend are deployed, we have to configure our requests URL. 
+To do this, we have to use a feature in Heroku called Heroku configuration files or Heroku secrets. Basically, this is a way for us
+to write down important information such as our backend request route in a secure and inaccessible way. To do this we 
+have to navigate back to our frontend project. Now we run the command:
+```shell
+heroku config:set REQUEST_URL=<backend_url>
+```
+
+This creates a configuration variable called REQUEST_URL which maps to the value of our backend url. Now we have to go to our React application and 
+replace where we write <strong> REACT_APP_API_DOMAIN </strong> with <strong> process.env.REQUEST_URL </strong>
+
+Therefore, everytime we make a request, our React application will look for our current processes configuration variable called REQUEST_URL, which Heroku 
+stored, and make requests using that URL.
+
+Now that we changed our API request URL we have to re-deploy our frontend with this modified changes. To do this,
+we simply navigate back to our frontend project directory and type in:
+```shell
+git add .
+```
+Which adds all our modified files.
+
+```shell
+git commit -m "<message>" .
+```
+
+Which commits our newly updated code.
+
+```shell
+git push heroku master
+```
+
+Will push the new changes to Heroku and update our current project.
+
+Now, our frontend application will make requests to our new request URL. 
+
+### Deploy PostgreSQL
+Before we proceed in deploying our database we have to note one file. Under our <strong> service/src/main/java/com/codeforcommunity/util</strong> file you will see a class called PropertiesLoader.
+This class will examine our resources file under <strong> persist/src/main/resources </strong> and get our database properties. This is useful because this is how our backend service knows where 
+to find our PostgreSQL database. 
```

Comment:
Maybe put this with the comment above where I originally wanted you to work on this since it also provides the port to start up on